src/main/blackbox/blackbox.c

```diff
@@ -213,7 +213,7 @@ static const blackboxDeltaFieldDefinition_t blackboxMainFields[] = {
 #ifdef PITOT
     {"AirSpeed",   -1, SIGNED,   .Ipredict = PREDICT(0),       .Iencode = ENCODING(SIGNED_VB),   .Ppredict = PREDICT(PREVIOUS),      .Pencode = ENCODING(TAG8_8SVB), FLIGHT_LOG_FIELD_CONDITION_PITOT},
 #endif
-#ifdef SONAR
+#ifdef RANGEFINDER
```

Comment:
If we are changing the build flag, then I would prefer `USE_RANGEFINDER`. Eventually I'd like to see all the build flag converted to a `USE_` prefix.